python/ray/data/_internal/execution/operators/hash_shuffle.py

```diff
@@ -445,12 +446,152 @@ def __init__(
             int, Dict[int, _PartitionStats]
         ] = defaultdict(dict)
 
+        # Resource monitoring state
+        self._aggregator_startup_time: Optional[float] = None
```

Comment:
```suggestion
        self._started_at: Optional[float] = None
```

Reply:
nit: This isn't really an aggregator, and time ref is confusing (it reads as the amount of time it took to startup)